lib/tasks/queue.rake

```diff
@@ -40,4 +40,19 @@ namespace :queue do
 
     RequeueContent.new(scope).call
   end
+
+  desc "Preview of the message published onto rabbit MQ"
+  task :preview_recent_message, [:document_type] => :environment do |_, args|
+    document_type = args[:document_type]
+    raise ValueError("expecting document_type") unless document_type.present?
+
+    scope = Edition
+              .with_document
+              .where(editions: { state: 'published' })
+              .where(document_type: document_type)
+              .order(public_updated_at: :desc)
+              .limit(1)
+
+    pp RequeueContent.new(scope).preview
```

Comment:
I don't think the preview functionality should be the responsibility of the `RequeueContent` class since that's just concerned about requeueing all the content, not everything related to the queue. I would be tempted to include the lines which call the DownstreamPayload into this task directly and leave `RequeueContent` as it was.

Reply:
@thomasleese Have moved all the code to the rake task and added a test to ensure it is updated if the code changes